wordpress-mobile/WordPress-iOS
# "Activate" vs "Active" Wording on Theme Customization

Using `theme.isCurrentTheme` to check whether wording on menu bar for "Try and Customize" theme screen should be "Activate" or "Active" instead of always using "Active".

Fixes #17060

## Screenshot
https://user-images.githubusercontent.com/13263478/131718289-7a3fcc9d-6819-45b5-856e-8faa5423bac9.mp4

## Testing
To test:
1. Go to Themes
2. Click menu bar for an **inactive** theme
3. Choose "Try and Customize" on bottom sheet
4. Notice that top right menu bar button should say "Activate"

1. Go to Themes
2. Click menu bar for **active** theme
3. Choose "Try and Customize" on bottom sheet
4. Notice that top right menu bar button should say "Active"

## Regression Notes
1. Potential unintended areas of impact
Using "Activate" on active themes.

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Tested active theme's customization screen as well.

## PR submission checklist:
